#request
int8 my_request
---
#response
int8 my_response